src/mocks.ts

```diff
@@ -0,0 +1,3 @@
+import { FilterType } from './types/FilterType';
+
+export const FILTERS: FilterType[] = ['All', 'Active', 'Completed'];
```

Comment:
You can create type from an array
https://stackoverflow.com/questions/59857223/how-to-convert-typescript-types-of-strings-to-array-of-strings